download-ecosystem-tools.md

```diff
@@ -46,7 +46,7 @@ Depending on which tools you want to use, you can install the corresponding offl
 | [TiDB Data Migration (DM)](/dm/dm-overview.md)  | `dm-worker-{version}-linux-{arch}.tar.gz` <br/>`dm-master-{version}-linux-{arch}.tar.gz` <br/>`dmctl-{version}-linux-{arch}.tar.gz`  |
 | [TiCDC](/ticdc/ticdc-overview.md)  | `cdc-{version}-linux-{arch}.tar.gz`  |
 | [Backup & Restore (BR)](/br/backup-and-restore-overview.md)  | `br-{version}-linux-{arch}.tar.gz`  |
-| [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md)  | `sync_diff_inspector`  |
+| [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md)  | For TiDB v9.0.0 and later versions: included in TiUP and `tiflow-{version}-linux-{arch}.tar.gz` <br/>For TiDB versions before v9.0.0: `sync_diff_inspector` from tidb-tools  |
```

Comment:
This line looks confusing. It might cause the misunderstanding that other tools are not included in tiup 

![image](https://github.com/user-attachments/assets/cd90c11d-1fa8-491f-84c8-117f2ba393c2)

```suggestion
| [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md)  | For TiDB v9.0.0 and later versions: included in `tiflow-{version}-linux-{arch}.tar.gz` <br/>For TiDB versions before v9.0.0: `sync_diff_inspector` from tidb-tools  |
```